documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -9,14 +9,22 @@ Define a service monitor instances that configures Prometheus to monitor your
 
 .Procedure
 
-. Create an authentication secret that contains {brandname} credentials so that Prometheus can authenticate with your {brandname} cluster.
+. Retrieve the credentials for {ispn_operator}.
 +
 [source,options="nowrap",subs=attributes+]
 ----
 include::yaml/basic_auth.yaml[]
 ----
 +
-<1> Specifies an application user. `developer` is the default.
+. Create an authentication secret that contains the credentials. +
+This allows Prometheus to authenticate with your {brandname} cluster and retrieve metrics.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/basic_auth.yaml[]
+----
++
+<1> Specifies the **operator** user.
```

Comment:
@ryanemerson Do you think users might object to re-using the operator credentials for Prometheus authentication? It's convenient but maybe more security-minded users would prefer to keep credentials strictly separate. Just a thought. I'm not sure if it's possible to add user-defined admin credentials.

Reply:
> Do you think users might object to re-using the operator credentials for Prometheus authentication?

IMO no, as the k8s users require a certain level of permissions to access the operator secrets in the first place. If a user has that level of access, they can do far worse things (e.g. delete clusters) then configure prometheus :fire: .